java/src/main/java/ai/onnxruntime/TensorInfo.java

```diff
@@ -251,7 +251,7 @@ public static TensorInfo constructFromBuffer(Buffer buffer, long[] shape, OnnxJa
 
     long elementCount = OrtUtil.elementCount(shape);
 
-    long bufferCapacity = buffer.capacity();
+    long bufferCapacity = buffer.remaining();
```

Comment:
>bufferCapacity [](start = 9, length = 14)

Suggest to rename it? spaceRemaining??

Reply:
Done.

Reply:
I picked `bufferRemaining`, because there could be more space available, it's just up until the current limit set on the buffer. Probably could be `bufferElementsRemaining` too, but the other one is shorter.